test/integration/auth/node_test.go

```diff
@@ -248,6 +275,9 @@ func TestNodeAuthorizer(t *testing.T) {
 	expectAllowed(t, createNode2(node2Client))
 	expectAllowed(t, updateNode2Status(node2Client))
 	expectAllowed(t, deleteNode2(node2Client))
+	// node evicting pod bound to itself is allowed
+	expectAllowed(t, createNode2MirrorPod(node2Client))
+	expectAllowed(t, createNode2MirrorPodEviction(node2Client))
```

Comment:
Rewrote the integration tests in 6237c8e89b05592bc4a03bc0453bda28ce2adc58 with your comments in mind. I hope they express the expected results for each action more clearly.